### PR TITLE
[UI] Update some links

### PR DESCRIPTION
**Pull request by chrysle**
Replace links in the sidebar pointing to backlink pages with a separately created page so that every visitor has access to them and the displayed text can be amended. Also merge the two categories for extendable and incorrect articles into one.

### CODE REVIEW

File: inyoka/wiki/locale/de_DE/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -676,3 +673,6 @@ msgstr "Seiten"
 msgid "“%(title)s” was reverted successfully"
 msgstr "Die Seite „%(title)s“ wurde erfolgreich wiederhergestellt."
 
+#~ msgid "Expandable Articles"
+#~ msgstr "Ausbaufähige Artikel"
+
```
Comment (chris34) [inline inyoka/wiki/locale/de_DE/LC_MESSAGES/django.po:678]: as you deleted the only occurrence in the HTML, you can also delete the commented translation (in the worst case we have the git log)

```suggestion

```

File: inyoka/portal/locale/de_DE/LC_MESSAGES/django.po
Comment (chris34) [inline inyoka/portal/locale/de_DE/LC_MESSAGES/django.po:1]: I'm a bit surprised that basically the whole file was commented. That would mean, the strings are not used. However, they are.

Just as an idea for a cause: Did you check, if your `development_settings` fits to the `example_development_settings`?

**Review by chris34 — COMMENTED** [review]
see the inline comments

 - [ ] Compiling the translation is missing. From https://doc.inyokaproject.org/getting_started.html#translate-inyoka:

> If the translations are done, run the following command to compile the corresponding ``*.mo`` files (binary translation files)
>
>          (inyoka)$ python manage.py compilemessages
>
> and restart the server for testing.
>
> It is recommended to add the ``*.mo`` files in a seperate commit, because they cannot
be merged by git. In case of a merge conflict, the commit can be dropped, the ``*.po`` files merged
and the ``*.mo`` files compiled again.

 - Lastly, a more an organizational question: When should i merge the changes?
As old and new pages don't display the same content at the moment.

File: inyoka/portal/locale/de_DE/LC_MESSAGES/django.po
Comment (chrysle) [reply inyoka/portal/locale/de_DE/LC_MESSAGES/django.po:1]: Not sure what could have been the cause; I removed the comments and re-created the message files, then it worked.

**Comment by chrysle** [comment]
> Compiling the translation is missing. From https://doc.inyokaproject.org/getting_started.html#translate-inyoka:

Oh, sorry, I must admit I'm not too familiar with gettext. Done.

>   Lastly, a more an organizational question: When should i merge the changes?
>    As old and new pages don't display the same content at the moment.

Do you mean that change should be publicly announced?

**Comment by chris34** [comment]
> Do you mean that change should be publicly announced?

That was not really my intention by the comment. I wanted to know if the new wiki page is already ready. Or if there will be more tweaking with the MetaFilter, before i should merge this PR.

Regarding the announcement however, feel free to add at least a line to the change log for the next release in this PR. Then it's more likely it will be mentioned at least in the uu.de's UWR. That's IMO enough for this change.

> It would be good if you could render all pages with the extendable template manually beforehand, so that the tags are added.

After each deployment (and every night) all pages are rendered. Nevertheless, this will only affect the HTML, not the metadata.

An example: If we change the rendered HTML of headings in InyokaMarkup, that's already covered. In contrast, metadata AFAIR is only updated, when a new revision is saved.

I can try in a new PR to add metadata regeneration to the existing mechanism.

**Comment by chrysle** [comment]
> > I can try in a new PR to add metadata regeneration to the existing mechanism.
> 
> see #1367

Great, thanks for doing this! Then I assume this can be merged; the page is ready.

**Pull request merged**